helmfile.d/snippets/defaults.yaml

```diff
@@ -143,24 +150,113 @@ environments:
           prometheus-blackbox-exporter: {}
           prometheus:
             enabled: false
+            remoteWrite:
+                enabled: false
+            otomiThanos: false
             replicas: 1
+            role: standalone
             scrapeInterval: 60s
             retentionSize: 4GB
             storageSize: 5Gi
           redis-shared:
             enabled: false
+          thanos:
+            enabled: false
+            bucketweb:
+              resources:
+                requests:
+                  cpu: 100m
+                  memory: 128Mi
+                limits:
+                  cpu: 500m
+                  memory: 256Mi
+            query:
+              replicaCount: 1
+              resources:
+                requests:
+                  cpu: 200m
+                  memory: 128Mi
+                limits:
+                  cpu: '1'
+                  memory: 1Gi
+            queryFrontend:
+              resources:
+                requests:
+                  cpu: 200m
+                  memory: 128Mi
+                limits:
+                  cpu: 500m
+                  memory: 512Mi
+            receiver:
+              mode: standalone
+              replicaCount: 1
+              persistence:
+                size: 8Gi
+              resources:
+                requests:
+                  cpu: 200m
+                  memory: 128Mi
+                limits:
+                  cpu: '1'
+                  memory: 1Gi
+            storegateway:
+              persistence:
+                size: 8Gi
+              resources:
+                requests:
+                  cpu: 200m
+                  memory: 128Mi
+                limits:
+                  cpu: 500m
+                  memory: 256Mi
+            compactor:
+              resources:
+                requests:
+                  cpu: 200m
+                  memory: 128Mi
+                limits:
+                  cpu: 500m
+                  memory: 256Mi
+              retentionResolutionRaw: 30d
+              retentionResolution5m: 30d
+              retentionResolution1h: 10y
+              persistence:
+                size: 8Gi
```

Comment:
> The compactor needs local disk space to store intermediate data for its processing as well as bucket state cache. Generally, for medium sized bucket about 100GB should be enough to keep working as the compacted time ranges grow over time.
https://thanos.io/tip/components/compact.md/#disk

8 GB does not seem to be a good default for compactor